src/lib/annotations/Annotator.js

```diff
@@ -663,6 +673,7 @@ class Annotator extends EventEmitter {
             });
         } else if (mode === TYPES.draw) {
             const drawingThread = this.createAnnotationThread([], {}, TYPES.draw);
```

Comment:
I don't think that keeps in line with the pending -> commit pattern. But I am creating a new DrawingController class to manage threads separately. And developers will be able to plug in their own controllers.